docker-compose.yml

```diff
@@ -27,9 +26,34 @@ services:
       # existing content of the image at the same location
       - /app/public/sites
     restart: always
+    environment:
+      FRANKENPHP_CONFIG: "worker ./public/index.php"
+      SERVER_NAME: :80
```

Comment:
Can you remove these labels please? We try to keep these example repositories as minimal as possible.

Reply:
I'll move them to another branch on our fork.